pkg/apis/rook.io/v1alpha2/types.go

```diff
@@ -60,13 +60,14 @@ type Directory struct {
 type Selection struct {
 	// Whether to consume all the storage devices found on a machine
 	UseAllDevices *bool `json:"useAllDevices,omitempty"`
-
 	// A regular expression to allow more fine-grained selection of devices on nodes across the cluster
 	DeviceFilter string `json:"deviceFilter,omitempty"`
-
+	// List of devices to use as storage devices
 	Devices []Device `json:"devices,omitempty"`
-
+	// List of host directories to use as storage
 	Directories []Directory `json:"directories,omitempty"`
+	// PersistentVolumeClaims to use as storage
+	VolumeClaimTemplates []v1.PersistentVolumeClaim `json:"volumeClaimTemplates,omitempty"`
```

Comment:
How do storage providers configure the storage on the volume claims? Are they expected to behave the same as the `directories` when they are mounted in the pod?